cpp/include/cudf/table/row_operators.cuh

```diff
@@ -462,18 +462,20 @@ template <template <typename> class hash_function, bool has_nulls = true>
 class row_hasher {
  public:
   row_hasher() = delete;
-  row_hasher(table_device_view t) : _table{t} {}
+  row_hasher(table_device_view t) : _table{t}, _seed(DEFAULT_HASH_SEED) {}
+  row_hasher(table_device_view t, uint32_t seed) : _table{t}, _seed(seed) {}
 
   __device__ auto operator()(size_type row_index) const
   {
-    auto hash_combiner = [](hash_value_type lhs, hash_value_type rhs) {
-      return hash_function<hash_value_type>{}.hash_combine(lhs, rhs);
+    auto hash_combiner = [=](hash_value_type lhs, hash_value_type rhs) {
+      return hash_function<hash_value_type>{_seed}.hash_combine(lhs, rhs);
     };
 
     // Hashes an element in a column
     auto hasher = [=](size_type column_index) {
       return cudf::type_dispatcher(_table.column(column_index).type(),
-                                   element_hasher<hash_function, has_nulls>{},
+                                   element_hasher_with_seed<hash_function, has_nulls>{
+                                     _seed, std::numeric_limits<hash_value_type>::max()},
```

Comment:
I don't think this is what you want. This incorporates the seed into each column. I'd think the seed should just be the initial value in the `transform_reduce` below.